PWGHF/TableProducer/candidateCreator2Prong.cxx

```diff
@@ -210,25 +234,167 @@ struct HfCandidateCreator2Prong {
     }
   }
 
-  void processPvRefit(aod::Collisions const& collisions,
-                      soa::Join<aod::Hf2Prongs, aod::HfPvRefit2Prong> const& rowsTrackIndexProng2,
-                      aod::TracksWCov const& tracks,
-                      aod::BCsWithTimestamps const& bcWithTimeStamps)
+  template <bool doPvRefit = false, typename Cand, typename TTracks>
```

Comment:
```suggestion
  template <bool doPvRefit, typename Cand, typename TTracks>
```
It is more clear in the code when the template parameter is provided for both cases.